### DISCUSSION TITLE
could not satisfy dependencies: - removing chromium breaks dependency 'chromium' required by flaresolverr

### DISCUSSION THREAD

**Discussion by IAmHugh001** [FlareSolverr/FlareSolverr#701] [Q&A]
### Have you checked our README?

- [X] I have checked the README

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you checked the discussions?

- [X] I have read the Discussions

### Environment

```markdown
- FlareSolverr version: 3.0.2
- Last working FlareSolverr version: 3.0.2
- Operating system: Garuda and Arch (installed using the Calam installer)
- Are you using Docker: no, never
- FlareSolverr User-Agent (see log traces or / endpoint): no idea where these logs are
- Are you using a VPN: no
- Are you using a Proxy: no
- Are you using Captcha Solver: [yes/no]
- If using captcha solver, which one:
```

### Description

Not so much a bug as a question. Why is Chromium necessary for Flaresolverr?

### Logged Error Messages

```text
could not satisfy dependencies:
- removing chromium breaks dependency 'chromium' required by flaresolverr
```

**Comment by ilike2burnthing** [comment]
The challenge page is loaded in Chromium.

**Comment by txtsd** [comment]
FlareSolverr comes with a chromium driver, which uses the chromium on your system.

**Comment by ngosang** [comment]
With this AUR package you don't have to install Chrome/Chromium.
https://aur.archlinux.org/packages/flaresolverr-bin
